bank/services/stoneBaas/stoneBaas.go

```diff
@@ -1,14 +1,118 @@
 package stoneBaas
 
 import (
+	"fmt"
 	"html/template"
+	"net/http"
 	"strconv"
 	"strings"
+	"sync"
 
+	"github.com/PMoneda/flow"
+	"github.com/mundipagg/boleto-api/certificate"
 	"github.com/mundipagg/boleto-api/config"
+	"github.com/mundipagg/boleto-api/log"
+	"github.com/mundipagg/boleto-api/metrics"
+	"github.com/mundipagg/boleto-api/models"
 	"github.com/mundipagg/boleto-api/tmpl"
+	"github.com/mundipagg/boleto-api/util"
+	"github.com/mundipagg/boleto-api/validations"
 )
 
+var (
+	onceTransport = &sync.Once{}
+	transportTLS  *http.Transport
+)
+
+type bankStoneBaas struct {
+	validate    *models.Validator
+	log         *log.Log
+	trasportTLS *http.Transport
+}
+
+func New() (bankStoneBaas, error) {
+	var err error
+	b := bankStoneBaas{
+		validate: models.NewValidator(),
+		log:      log.CreateLog(),
+	}
+
+	b.configureMTLS(&err)
+
+	if err != nil {
+		return bankStoneBaas{}, fmt.Errorf("fail on load TLSTransport: %v", err)
+	}
+
+	b.pushRequestValidations()
+
+	return b, nil
+}
+
+func (b bankStoneBaas) ProcessBoleto(boletoRequest *models.BoletoRequest) (models.BoletoResponse, error) {
+	errs := b.ValidateBoleto(boletoRequest)
+
+	if len(errs) > 0 {
+		return models.BoletoResponse{Errors: errs}, nil
+	}
+
+	return b.RegisterBoleto(boletoRequest)
+}
+
+func (b bankStoneBaas) RegisterBoleto(boletoRequest *models.BoletoRequest) (models.BoletoResponse, error) {
+	var res string
+	var statusCode int
+	var resHeaders map[string]interface{}
+	var err error
+
+	baasUrl := config.Get().BaasUrl
+	timeout := config.Get().TimeoutRegister
```

Comment:
Opa, como temos vários lugares(possibilidades) de passar timeouts, dai vale deixar essa var com o nome mais explicito 😀 
```suggestion
	timeoutRegister := config.Get().TimeoutRegister
```